models/marts/fact_restaurant_feedback.sql

```diff
@@ -0,0 +1,13 @@
+{{ config(materialized='table') }}
+
+SELECT 
```

Comment:
Can you leave me the mart dimensional modeling from `dbdiagram` please, maybe you should draw and update it on the PR description